kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -20,6 +20,8 @@ data:
           # If you want to continue to disable kube-system,gatekeeper-system log collection keep the namespaces in the following setting and add any other namespace you want to disable log collection to the array.
           # In the absense of this configmap, default value for exclude_namespaces = ["kube-system","gatekeeper-system"]
           exclude_namespaces = ["kube-system","gatekeeper-system"]
+          # If you want to collect logs from only selective pods inside system namespaces add them to the following setting. Provide namepace:controllerName of the system pod. NOTE: this setting is only for system pods
+          # collect_system_pod_logs = ["kube-system:coredns"]
```

Comment:
For this to work, system namespaces corresponding to the pods in collect_system_pod_logs  to be the exclude_namespaces list or not?  Instructions in the configmap not clear to me.

Reply:
include_system_pods only works if the system pod is in the exlude namespace list.